utilities/list-methods.metta

```diff
@@ -38,11 +38,32 @@
    (if (< $x $head)
        (Cons $x (Cons $head $tail))
        (Cons $head (List.insert $x $tail))))
+;; A function that converts a list to an expression
+; (: List.toExpr (->(List $a) Expression)
+(= (List.toExpr Nil) ())
```

Comment:
We have the same function with the name `List.listToExpr` and also another function named `exprToList` both doing exactly what you're doing.